tornado-api/src/main/java/uk/ac/manchester/tornado/api/memory/XPUBuffer.java

```diff
@@ -22,33 +22,33 @@
 import uk.ac.manchester.tornado.api.exceptions.TornadoMemoryException;
 import uk.ac.manchester.tornado.api.exceptions.TornadoOutOfMemoryException;
 
-public interface ObjectBuffer {
+public interface XPUBuffer {
```

Comment:
why xpu ? we dont reuse it in opencl?

Reply:
`ObjectBuffer` is too broad term. I wanted to refer to a Buffer on the accelerator. This interface is common for all backends, including OpenCL